src/test/test_utils.cpp

```diff
@@ -427,10 +427,20 @@ void assertStringResponse(const tensorflow::serving::PredictResponse& proto, con
 void assertStringResponse(const ::KFSResponse& proto, const std::vector<std::string>& expectedStrings, const std::string& outputName) {
     ASSERT_EQ(proto.outputs_size(), 1);
     ASSERT_EQ(proto.outputs(0).name(), outputName);
-    ASSERT_EQ(proto.outputs(0).datatype(), "BYTES");
-    ASSERT_EQ(proto.outputs(0).shape_size(), 1);
+    ASSERT_EQ(proto.outputs(0).datatype(), "UINT8");
```

Comment:
This assert should still check for BYTES datatype but `assertStringOutputProto(KFS)` should be changed to check for correct data in raw field.
Correct raw field meaning each batch is prepended with 4 bytes of unsigned integer which represents its length. Check other fixtures, the assert is probably already implemented